### ISSUE TITLE
print-pdf does not work correctly

### ISSUE THREAD

**Issue by Helias** [hakimel/reveal.js#3499]
Using the parameter `?print-pdf` does not work properly with these slides, the background becomes white

Expected behavior:
![image](https://github.com/hakimel/reveal.js/assets/519778/33cff86d-bef6-4bd2-9455-c84a1245dd60)

Current behavior:
![image](https://github.com/hakimel/reveal.js/assets/519778/5b3cb649-a4c8-4650-b945-07446528bd43)

[slides-code.html.zip](https://github.com/hakimel/reveal.js/files/13064508/slides-code.html.zip)

**Comment by Shashankpantiitbhilai** [comment]
Let me fix it,assign this to me

**Comment by t-fritsch** [comment]
Hi @Helias 

Maybe I'm wrong but isn't your deck created using https://slides.com/ ?

If that's the case I suggest you post your bug report in https://help.slides.com/

**Comment by Helias** [comment]
Yes it is, I am sorry, I will create an issue there but I was thinking that the issue depends on revealjs

I am sorry

**Comment by t-fritsch** [comment]
No problem Helias ! 
I downloaded your html and since it's compiled it's hard to tell what the bug is related to, so I'm guessing the problem lies in the way you add your background but not sure about that (in a bare reveal.js project the background when adding `?print-pdf` shows well : https://fcdc4r.csb.app/?print-pdf)

I'm closing this issue for now but if it turns out this is a bug of reveal don't hesitate to tell, we'll reopen this issue 🙏
